pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -41,8 +41,7 @@ type EventDispatcher interface {
 
 	// PostStopPod is invoked after all of a pod's containers have permanently
 	// stopped running, but before the pod sandbox is destroyed.
-	PostStopPod(pod *v1.Pod, cgroupPath string) (*lifecycle.EventReply, error)
-
+	PostStopPod(cgroupPath string) error
```

Comment:
I think sending the pod spec would make a lot of sense. Semantics of PostStopPod could be, the reply doesn't get written through to the VFS. So it could be passed to the isolator but only for informational purposes. It's cheap to do in terms of code anyway and helps to make the events more consistent.